flux/clusters/k8s-01/insights-api-db/overlays/PROD/postgrescluster-pgbackrest-adjustment.yaml

```diff
@@ -1,3 +1,7 @@
 - op: add
   path: /spec/backups/pgbackrest/global/archive-timeout
   value: '360'
+
+- op: add
+  path: /spec/patroni/dynamicConfiguration/postgresql/parameters/wal_level
```

Comment:
I've seen wal_level=minimal and somewhere nearby there's still pgbackrest trying to archive wals. Also I've seen replicas:2. Replicas aren't going to have any data on them if wal_level = minimal and thus master reelection is going to be a data loss. Do I get it right or is there something else preventing this?

Reply:
When wal_level is set to minimal it minimizes the amount of data written to WAL, which primarily supports crash recovery and physical replication, but not logical replication, yes. But there is no replicas:2 for postgres cluster, probably you have seen here: https://github.com/konturio/disaster-ninja-cd/blob/459570509cad8a7dd5759685e103ecda3d4cdf9b/helm/insights-api/values/values-prod.yaml#L10 (replicas of the app). 
Regadring pgbackrest: it can still archive WALs with wal_level=minimal.